pages/api/setRepoConfig.ts

```diff
@@ -23,35 +23,41 @@ const setRepoConfigHandler = async (
 	// check auth
 	const session = await getServerSession(req, res, authOptions);
 	if (!session || !session.user.id) {
+		const eventProperties = { response_status: 401 };
+        rudderStackEvents.track("absent", "", 'set-repo-config', { type: 'HTTP-401', eventStatusFlag: 0, eventProperties });
 		return res.status(401).json({ message: 'Unauthenticated' });
 	}
 
 	// validate request
 	if (req.method !== 'POST') {
+		const eventProperties = { response_status: 405 };
+        rudderStackEvents.track("absent", "", 'set-repo-config', { type: 'HTTP-405', eventStatusFlag: 0, eventProperties });
 		return res.status(405).json({ message: 'setRepoConfig API must be called using POST method' });
 	}
 	if (!isSetRepoConfigReqBody(req.body)) {
+		const eventProperties = { response_status: 400 };
+        rudderStackEvents.track("absent", "", 'set-repo-config', { type: 'HTTP-400', eventStatusFlag: 0, eventProperties });
 		return res.status(400).json({ message: 'setRepoConfig API received invalid values for one or more of its fields' })
 	}
 
 	// update repo config
 	const { repo, configType, value }: SetRepoConfigReqBody = req.body;
 	const eventProperties = {
 		...repo,
-		configType,
-		updatedValue: value
+		config_type: configType,
+		updated_value: value
 	}
 	await setRepoConfig(repo, session.user.id, configType, value)
 		.then((queryResponse) => {
 			if (queryResponse) {
-				rudderStackEvents.track(session.user.id!, "", 'settings-changed', { type: 'setting', eventStatusFlag: 1, name: getAuthUserName(session), eventProperties })
```

Comment:
Can we maintain a record of which tables are duplicate? `settings-changed` table is going to be in the database, which will not be updated anymore. 

Maybe just create a ticket for this (for either documenting this or for migrating the contents of the settings-changed table)

Low LOGAF

Reply:
created an issue with P2 priority in the project `Launch Madness`